### PR TITLE
Jakt fixes

### PR DESCRIPTION
**Pull request by alimpfard**
Unrelated Shell commit, plus fix for `serenity.sh recreate` deleting the jakt runtime with no way to get it back automatically.

### CODE REVIEW

File: Userland/Shell/main.cpp [hunk 1]
```diff
@@ -181,6 +181,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     parser.add_option(posix_mode, "Behave like a POSIX-compatible shell", "posix");
     parser.add_positional_argument(file_to_read_from, "File to read commands from", "file", Core::ArgsParser::Required::No);
     parser.add_positional_argument(script_args, "Extra arguments to pass to the script (via $* and co)", "argument", Core::ArgsParser::Required::No);
+    parser.add_ignored("login", 'l');
```
Comment (nico) [inline Userland/Shell/main.cpp:184]: Unrelated, put in separate commit?

Comment (alimpfard) [reply Userland/Shell/main.cpp:184]: Oh I don't actually want that in there, will remove.
